/mocap_data_convert:
  ros__parameters:
    pos_topic : "/vrpn_client_node/uav/pose"
    vel_topic  : "/vrpn_client_node/uav/twist"